.travis.yml

```
@@ -21,6 +21,8 @@ jobs:
       dist: xenial
     - stage: test
       python: '3.8'
+    - stage: test
+      python: '3.9'
```

You can test against `3.9-dev` whilst we're waiting for them to add `3.9`:

```suggestion
      python: '3.9-dev'
```

btw this is the tracking issue to +1
* https://travis-ci.community/t/python-3-9-0-build/10091

---

Often times dev and the final releases aren't quite the same. I think this happened recently with 3.8 and I'd rather not start testing against -dev and trying to fix those bugs

---

Sure.

On Travis, `-dev` is a daily build from the matching CPython branch.

On GitHub Actions, `-dev` is the last tagged release (e.g. alpha, beta, rc, final).

---

Thanks for the feedback guys. In hindsight I should've realized I could just add ```-dev```.

> btw this is the tracking issue to +1

From the looks of this travis will have support for 3.9 very soon, I will update with ```-dev``` in the mean time if you guys are ok with it